upstream.wrap

```diff
@@ -0,0 +1,7 @@
+[wrap-file]
+subdir = box2d-2.3.1
+
+source_url = https://codeload.github.com/erincatto/box2d/zip/v2.3.1
```

Comment:
I'd preffer to use https://github.com/erincatto/box2d/archive/v2.3.1/box2d-2.3.1.tar.gz

What do you think?